saltminion 3006.4 - Corrected url/64_py3 and checksum/64_py3

LeaDevelop opened the pull request:
Path to saltminion .exe on official saltstack repository has changed through time and chocolatey packages will result in failure to install. It makes sense to keep history version for 3006.x up to date as it is a LTS version.
https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-version-support-lifecycle.html#version-support-lifecycle

- I created directory and pulled chocolateyuninstall.ps1 script, it is untouched.
- chocolateyinstall.ps1 script got url corrections for both executables including the checksums
- saltminion.nuspc got url corrections for documentation and pointed to up to date official logo

LeaDevelop committed:
- nuspec iconUrl, docsUrl, releaseNotes url updated
- Corrected chocolateyInstall.ps1 url/64_py3 and checksum/64_py3 for saltminion 3006.4

twangboy commented on 3006-4/saltminion.nuspec:
Salt Project

twangboy commented on 3006-4/saltminion.nuspec:
www.saltproject.io

twangboy commented on 3006-4/saltminion.nuspec:
https://docs.saltproject.io/en/3006/contents.html

LeaDevelop committed:
Updated authors, project and docs URL

barkingfoodog commented:
Hi, thank you for the update! Would you please move the `tools/` directory one level higher and eliminate the `saltminion/` directory? This will allow the relative path in `saltminion.nuspec` to locate the files under `tools/`.

LeaDevelop committed:
3006.4 adjusted directory structure
- tools/ level higher
- saltminion/ eliminated
- moved out legacy (aligned with master branch)

LeaDevelop commented:
@barkingfoodog PR was aligned with master branch and suggested improvements done, ready for review
